src/main/java/ch3/Car.java

```diff
@@ -3,15 +3,40 @@
 public class Car {
 
     private int position = 0;
+    private String name;
 
-    public int getPosition() {
+    public int position() {
         return position;
     }
 
+    public String name() {
+        return name;
+    }
+
     public void move(int number) {
-        final int MOVE_CONDITION_NUMBER = 4;
-        if (number >= MOVE_CONDITION_NUMBER) {
+        if (isMovingCondition(number)) {
             this.position++;
         }
     }
+
+    private boolean isMovingCondition(int number) {
+        final int MOVE_CONDITION_NUMBER = 4;
+        return number >= MOVE_CONDITION_NUMBER;
+    }
+
+    public void makeName(String name) {
+        if (isNameMaxLength(name)) {
+            throw new RuntimeException("이름은 5자를 초과할 수 없습니다");
+        }
+        this.name = name;
+    }
+
+    private boolean isNameMaxLength(String name) {
+        final int MAX_NAME_LENGTH = 5;
+        return name.length() > MAX_NAME_LENGTH;
+    }
+
+    public boolean isWinningCar(int maxPosition) {
+        return maxPosition == position;
+    }
```

Comment:
`Winning`이라는 의미는 `CarRacing`의 책임과 관련있는 단어가 아닐까? 생각해봤어요.

지금 구조가 잘못되었다기 보다는, 책임을 분명히 하기위한 개인적인 관점으로 봐주셔도 좋습니다~

```suggestion
    public boolean 포지션_같은지_바교(int other) {
        return this.position == other;
    }
```